app-backend/db/src/main/resources/migrations/V16__add_geojson_filetype.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE file_type ADD VALUE 'GEOJSON' AFTER 'GEOTIFF_WITH_METADATA';
```

Comment:
separate migration because flyway complains about putting this type of operation with the others